app/services/force/listing_service.rb

```diff
@@ -33,6 +33,13 @@ def self.listing(id, opts = {})
       Rails.logger.info("Calling self.listing for #{id} with force: #{force}")
       results = Request.new(parse_response: true).cached_get(endpoint, nil, force)
       results_with_cached_listing_images = add_cloudfront_urls_for_listing_images(results)
+      # TODO: DAH-2636 conditionally get translations using feature flag
+      if ::UNLEASH.is_enabled? 'test'
+        puts 'test is enabled'
```

Comment:
thought: We can ignore the coverage issue and bypass it to merge this PR, but in the follow-up ticket we should update the specs for this method.